data/catalogs/deltares_data.yml

```diff
@@ -622,14 +565,16 @@ fabdem:
     paper_ref: Hawker et al. (2022)
     source_license: CC BY-NC-SA 4.0
     source_url: https://data.bris.ac.uk/data/dataset/25wfy0f9ukoge2gs7a5mqpq2j7
-    source_version: 1.0
+    source_spatial_extent:
+      West: -180.0
+      South: -60.0
+      East: 180.0
+      North: 80.0
+  version: 1.0
   path: topography/fabdem/fabdem.vrt
   rename:
     fabdem: elevtn
 
-gadm:
-  alias: gadm_level0
-
 gadm_level0:
```

Comment:
This would only shorten the file.
Given the ongoing discussions about placeholders, variants, etc I think it's perhaps better to not use placeholders in daltares_data.yml